src/fastcontext/asm.S

```diff
@@ -782,6 +786,119 @@ SET:
         blr
 #endif
 
+#ifdef NEEDARMA64CONTEXT
+#warning NEEDARMA64CONTEXT
+#warning GET SET
```

Comment:
Equally, see comment above.

Reply:
I'm not sure. I originally added it since the Armv7 code had it. I would agree it can be removed.

Reply:
Remove. If output for sanity is required, let's consider a print message at configure.

Reply:
Removed code. I pushed it to the PRDec Branch.